lib/capistrano/tasks/delayed_job.cap

```diff
@@ -0,0 +1,25 @@
+namespace :delayed_job do
+  desc "stop delayed job processes"
+  task :stop do
+    on roles(:app) do
+      within release_path do
+        execute "pwd"
+        env = "env RAILS_ENV=production"
+        num_jobs = fetch :num_delayed_job_processes
+        execute :rvm, "default do", env, "./bin/delayed_job -n", num_jobs, "stop"
```

Comment:
I don't love how I did this, but it wasn't working if I called it without the rvm nonsense.

Reply:
if methods work in .cap, this would be a nice use case!